### PR TITLE
feat(cucumber): add fullName and testCaseId props

### LINKED ISSUE #518: Test cases I upload with test results appear in launches/tree but not in test cases
**Describe the bug**
So as per the documentation, I ran a simple cucumber-js test and included and generated allure-results folder. I uploaded that folder in launches and launches listed it. I can see test cases under launches/tree but they do not appear under test cases.

**To Reproduce**
1. run the test with following reporter config

```
var CucumberJSAllureFormatter = require("cucumberjs-allure2-reporter").CucumberJSAllureFormatter;
var AllureRuntime = require("cucumberjs-allure2-reporter").AllureRuntime;
 
function Reporter(options) {
    CucumberJSAllureFormatter.call(this,
        options,
        new AllureRuntime({ resultsDir: "./allure-results" }),
        {
        labels: {
          epic: [/@feature:(.*)/],
          severity: [/@severity:(.*)/]
        },
        links: {
          issue: {
            pattern: [/@issue=(.*)/],
            urlTemplate: "http://localhost:8080/issue/%s"
          },
          tms: {
            pattern: [/@tms=(.*)/],
            urlTemplate: "http://localhost:8080/tms/%s"
          }
        }
      });
}
Reporter.prototype = Object.create(CucumberJSAllureFormatter.prototype);
Reporter.prototype.constructor = Reporter;
 
exports.default = Reporter;
```

2. Upload the allure-results to launches
3. Check launches/tree and test cases appear there.
4. Stop launch and visit test cases, the test cases do not appear.

**Expected behavior**
I want the test cases to appear under test cases once I upload automation results.

### PR DESCRIPTION
**Pull request by vovsemenv**
fixes #518

Full Name allow result to have historical data and create testcases in testops

### CODE REVIEW

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts [hunk 1]
```diff
@@ -286,12 +286,13 @@ export class CucumberJSAllureFormatter extends Formatter {
     const links = this.parseTagsLinks(scenario?.tags || []);
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
     const thread = data.workerId || ALLURE_THREAD_NAME || process.pid.toString();
-
     this.testCaseStartedMap.set(data.id, data);
     this.testCaseTestStepsResults.set(data.id, []);
     this.currentTestsMap.set(data.id, currentTest);
 
     currentTest.name = pickle.name;
+    currentTest.fullName = `${pickle.uri}#${pickle.name}`;
```
Comment (baev) [inline packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:294]: add `testCaseId` and `testCaseName` as well

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts [hunk 2]
```diff
@@ -289,10 +289,10 @@ export class CucumberJSAllureFormatter extends Formatter {
     this.testCaseStartedMap.set(data.id, data);
     this.testCaseTestStepsResults.set(data.id, []);
     this.currentTestsMap.set(data.id, currentTest);
-
+    const fullName = `${pickle.uri}#${pickle.name}`;
     currentTest.name = pickle.name;
-    currentTest.fullName = `${pickle.uri}#${pickle.name}`;
-    currentTest.testCaseId = `${pickle.uri}#${pickle.name}`;
+    currentTest.fullName = fullName;
+    currentTest.testCaseId = fullName;
```
Comment (baev) [inline packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:295]: md5()

**Review by baev — CHANGES_REQUESTED** [review]

**Review by epszaw — APPROVED** [review]
✅ The code looks nice. Guess nothing gonna be wrong.

**Review by baev — APPROVED** [review]

**Comment by baev** [comment]
@vovsemenv thanks!

**Pull request merged**